config/Dockerfile

```diff
@@ -1,5 +1,10 @@
 # NOTE: Keep this in sync with .ci-operator.yaml
-FROM registry.ci.openshift.org/openshift/boilerplate:image-v2.0.0
+# TODO: temporary until image is built, should be: FROM registry.ci.openshift.org/openshift/boilerplate:image-v3.0.0
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12
+
+# TODO: remove this COPY/RUN
+COPY build_image-v3.0.0.sh /build-temp.sh
+RUN /build-temp.sh
 
 COPY build.sh /build.sh
-RUN /build.sh && rm -f /build.sh
+RUN /build.sh
```

Comment:
I think we also want to consolidate the contents of build.sh into build_image-v3.0.0.sh and get rid of build.sh for this initial cut of v3.0.0.

Then, as you hint in your TODOs, cut a 3.0.1 with an empty build.sh that starts FROM 3.0.0. It'll be identical, but will get us back to where CI doesn't need to build the entire image from scratch, making runs faster.

Reply:
Makes sense, I was wondering why it had been split out - done!